Server/src/app.module.ts

```diff
@@ -5,11 +5,19 @@ import { AppService } from './app.service';
 import { LocationModule } from './location/location.module';
 import config from 'ormconfig';
 import { Location } from './location/entities/location.entity';
+import { ConfigModule } from '@nestjs/config';
+import configuration from 'config/configuration';
 
 @Module({
   imports: [
     TypeOrmModule.forRoot(config),
     LocationModule,
+    ConfigModule.forRoot({
+      envFilePath: `${process.cwd()}/config/env/${process.env.NODE_ENV}.env`,
```

Comment:
There should be a README instruction on how to set up these .env files!

Reply:
added a new PR (#27) with setup guide in README and added new script @mblomdahl